### PR TITLE
Export Several Helper Functions for Address Calculation

### PR DESCRIPTION
**Pull request by WGB5445**
### Description

Export Helper Functions

- createObjectAddress
- createResourceAddress
- createTokenAddress

### CODE REVIEW

**Review by 0xmaayan — COMMENTED** [review]
Please update CHANGELOG

**Comment by WGB5445** [comment]
> Please update CHANGELOG

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 
 # Unreleased
 
+- Export the functions: `createObjectAddress` `createResourceAddress` `createTokenAddress`
```
Comment (0xmaayan) [inline CHANGELOG.md:7]: we actually export everything under the `account` folder, right?

Comment (WGB5445) [reply CHANGELOG.md:7]: Yes, the account folder only contains these functions, and they will be used frequently.

Comment (WGB5445) [reply CHANGELOG.md]: I added the account folder to the changelog, so it may be more accurate.

**Review by 0xmaayan — APPROVED** [review]

**Pull request merged**